project/AppDependencies.scala

```diff
@@ -27,7 +27,8 @@ object AppDependencies {
       "com.github.fge"           % "json-schema-validator"  % "2.2.6",
       "org.json4s"              %% "json4s-native"          % "3.5.5",
       "org.mockito"              % "mockito-all"            % "1.10.19",
-      "org.scalacheck"          %% "scalacheck"             % "1.14.3"
+      "org.scalacheck"          %% "scalacheck"             % "1.14.3",
+      "com.github.tomakehurst"   % "wiremock"               % "2.15.0"
```

Comment:
Is there a reason we arent using the latest version of wiremock?

Reply:
bumped to the latest version